AsyncDisplayKit/ASDisplayNode.h

```diff
@@ -252,6 +257,17 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (ASLayout *)measureWithSizeRange:(ASSizeRange)constrainedSize;
 
+
+/**
+ * @abstract Provides a way to declare a block to provide an ASLayoutSpec without having to subclass ASDisplayNode and
+ * implement layoutSpecThatFits:
+ *
+ * @return The block used to provide a ASLayoutSpec
+ *
+ * @warning Overwriting layoutSpecThatFits: in a subclass and providing a layoutSpecBlock block is currently not supported
```

Comment:
Same thing.  For the @return, maybe "A block that takes a constrainedSize ASSizeRange argument, and must return an ASLayoutSpec that includes all of the subnodes to position in the layout.  This input-output relationship is identical to the subclass override method -layoutSpecThatFits:"